.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,39 @@
+name: Go
```

Comment:
Name could more descriptive, something like "Building and testing against different environments".